src/stdatamodels/jwst/datamodels/photom.py

```diff
@@ -18,7 +20,42 @@
 ]
 
 
-class FgsImgPhotomModel(ReferenceFileModel):
+class _PhotomModel(ReferenceFileModel):
```

Comment:
This class appears to be causing docs build failures. It might need to be added into the nitpick ignores, or else not be underscore-protected